clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAO.scala

```diff
@@ -84,7 +84,7 @@ abstract class PersistenceDAO(recoveryParallelism: Int) extends LazyLogging {
 
     val upsertId = ElasticsearchIndex.getUpsertId(document)
 
-    val jsonString = ModelAutoDerivation.defaultPrinter.pretty(document)
+    val jsonString = ModelAutoDerivation.defaultPrinter.print(document)
```

Comment:
same as above, although it looks like `document` and `defaultPrinter` now have different functions to achieve `pretty()`?

Reply:
Yep